src/core.jl

```diff
@@ -82,31 +82,64 @@ function add_vertices!(g::SimpleGraph, n::Integer)
 end
 
 """Return true if the graph `g` has an edge from `src` to `dst`."""
-has_edge(g::SimpleGraph, src::Int, dst::Int) = has_edge(g,Edge(src,dst))
+has_edge(g::SimpleGraph, src::Int, dst::Int) = has_edge(g, Edge(src,dst))
 
-"""Return an Array of the edges in `g` that arrive at vertex `v`."""
-in_edges(g::SimpleGraph, v::Int) = [Edge(x,v) for x in badj(g,v)]
-"""Return an Array of the edges in `g` that emanate from vertex `v`."""
-out_edges(g::SimpleGraph, v::Int) = [Edge(v,x) for x in fadj(g,v)]
+"""
+    in_edges(g, v)
+
+Returns an Array of the edges in `g` that arrive at vertex `v`.
+For undirected graphs each returned edge `e` will be ordered,
+i.e. src(e) ≤ dst(e).
+"""
+in_edges(g::DiGraph, v::Int) = [Edge(x,v) for x in badj(g, v)]
+in_edges(g::Graph, v::Int) = [x<=v ? Edge(x, v) : Edge(v, x) for x in badj(g, v)]
```

Comment:
if we *are* going to do this, we should use `is_ordered` here. But I don't think we should be doing this.